src/GTP.cpp

```diff
@@ -494,6 +550,10 @@ void GTP::execute(GameState & game, const std::string& xinput) {
                 }
             }
         }
+        if (!process_analyze_tags(id, cmdstream, game)) {
```

Comment:
I think you can attempt to parse tags in both the above "syntax not understood" failure branches. (again, to allow transitioning to passing the interval as a tag)